package.json

```diff
@@ -22,63 +22,49 @@
     "webpack-dev-server": "webpack-dev-server",
```

Comment:
Can you revert this file to the format it was before? I like to have a visual hierarchy for the commands since there are so damn many.